src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__link {
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav {
+    display: flex;
+    align-items: center;
+    height: 60px;
+
+    &-list {
+      display: flex;
+      list-style: none;
+      padding: 0;
+      height: 100%;
+    }
+
+    &-item {
+      position: relative;
+      height: 100%;
+      margin: 0 10px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use shorthand properties whenever possible. Here, `margin: 0 10px;` can be used as `margin: 0 10px;` without specifying `height: 100%;` again since it's already declared in line 27.